docs/status.md

```diff
@@ -1,20 +1,41 @@
 # Project status
 
-We’re really early on with the project and as always with any young project there are probably some sharp corners. But with the help of the open source community we plan to iron out those in the coming months.
+To gauge the level of interest in running k0s control planes as Kubernetes
+resources, we are releasing k0smotron at the earliest possible moment. Of
+cousre, as with any young project just getting started, there are bound to be
+some bumps, unforeseen issues, and a few surprises. In line with that, we can
+offer no guarantees regarding backward or forward compatibility, and you should
+expect a certain amount of breakage with each ensuing k0smotron release.
+
+All caveats aside, though, we are truly excited about what k0smotron has to
+offer, and look forward to seeing how it can transform the way you manage your
+Kubernetes deployments. We are confident that the open source community will
+take great interest in the project, too, and help us to smooth out the rough
+edges in the days, weeks, and months ahead.
 
-At this point we can not give much guarantees over backwards and forwards compatibility so expect things to break a bit with upcoming releases of k0smotron.
+## Cluster API
 
-We wanted to release k0smotron as early as possible to see whether there is in general interest of running k0s control planes as Kubernetes resources. We are excited about the potential that k0smotron brings to the table, and we look forward to seeing how it can transform the way you manage your Kubernetes deployments.
+One key focus of the k0smotron project is to get it to function as a Cluster
+API provider for both `ControlPlane` and worker `Bootstrap` providers. Once
+this goal is achieved, you will be able to use the Cluster API to provision the
+control plane (within the management cluster) and the worker nodes in your
+favourite infrastructure supporting cluster API.
 
+## Known limitations
 
-## Cluster API
+Areas in which k0smotron is currently limited include:
 
-One of the directions we're looking at is the have k0smotron working as a Cluster API provider for both `ControlPlane` and worker `Bootstrap` providers. What this means is that you could utilize Cluster API to provision both the control plane, within the management cluster, and worker nodes in your favourite infrastructure supporting cluster API.
+* Control Plane configurability
 
-## Known limitations
+    The configurability of the [k0s](https://docs.k0sproject.io/stable/) control plane is not yet fully enabled.
+
+* Control plane exposure
+
+    Currently k0smotron only supports `NodePort` and `LoadBalancer` services,
```

Comment:
Now it's `ClusterIP`, `NodePort`, and `LoadBalancer`. But everything else is true.